src/psyclone/psyGen.py

```diff
@@ -5446,6 +5446,10 @@ class Symbol(object):
                            datatype of the constant value must be \
                            compatible with the datatype of the symbol.
     :type constant_value: int, str or bool
+    :param precision: The amount of storage required by the datatype (bytes) \
+            or a reference to a Symbol holding the type information \
+            or a label identifying a default precision.
+    :type precision: int or :py:class:`psyclone.psyGen.Symbol` or str
```

Comment:
Are these the correct types? What about Symbol.Precision? Can it be str or int at the moment?

Reply:
Oh yes, apologies.